java/server/src/org/openqa/selenium/docker/Docker.java

```diff
@@ -93,7 +93,7 @@ public Image pull(String name, String tag) {
       throw new WebDriverException("Unable to pull container: " + name);
     }
 
-    LOG.info(String.format("Pull of %s:%s complete", name, tag));
+    LOG.fine(String.format("Pull of %s:%s complete", name, tag));
```

Comment:
Waiting for the pull takes a long time. This message informs the user that at least one of the images being pulled is available. Please leave.